Tests/ProgressKitTests/ProgressKitTests.swift

```diff
@@ -0,0 +1,16 @@
+import XCTest
+@testable import ProgressKit
+
+class ProgressKitTests: XCTestCase {
+    func testExample() {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct
+        // results.
+        XCTAssertTrue(true)
+    }
+
+
+    static var allTests = [
+        ("testExample", testExample),
```

Comment:
This is an artifact of the template test class Xcode makes.  This can be deleted.